tests/unit/Entity/FilesetTest.php

```diff
@@ -52,17 +57,17 @@ protected function setUp()
      */
     public function testAddFileToFileset()
```

Comment:
Do we want test for all the other functions? If you look at my PR, maybe we can merge some of the functionality?

Reply:
I added those tests for the other getters